lib/data/models/event_model.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:equatable/equatable.dart';
+
+class EventModel extends Equatable {
+  final String uid;
+  final String name;
+  final String cover;
+  final String status; // Offline/ Online Event
+  final String certificate;
+  final int date;
+  final String latitude; //used to add [Navigate to Venue]
```

Comment:
lat, long can be nullable if it is online event

Reply:
Done @Harryalways317. Will make the neccessary updates.